pymoose/pymoose/computation/utils.py

```diff
@@ -84,7 +84,7 @@ def serialize_computation(computation):
 
 
 def deserialize_computation(bytes_stream):
-    computation = msgpack.unpackb(bytes_stream, object_hook=_decode)
+    computation = msgpack.unpackb(bytes_stream, object_hook=_decode, raw=False)
```

Comment:
`False` is the default, or does this change anything?

Reply:
this being the default depends on the version, it wasn't working if I didn't make this explicit

Reply:
"wasn't working" as in the objects weren't being decoded properly, everything came out as bytes